app/editor/src/features/content/form/constants/timeLogColumns.tsx

```diff
@@ -5,31 +5,36 @@ import { Column } from 'react-table';
 import { ITimeTrackingForm } from '../components/time-log/interfaces';
 import { IContentForm } from '../interfaces';
 
+const colWidth = window.innerWidth > 1500 ? 140 : 100;
```

Comment:
Can we do this with a media query?

Reply:
I don't think so, because the class is generated by the react-table component, and I could not intercept it (and yes that was my first approach).